package.json

```diff
@@ -2,16 +2,20 @@
   "name": "poolifier",
   "version": "1.2.1",
   "description": "Library on top of node js worker threads that implement various worker pools type",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "npm install",
-    "test": "standard && nyc mocha --exit --timeout 20000 tests/*test.js ",
-    "debug-test": "mocha --inspect-brk --exit tests/*test.js ",
-    "demontest": "nodemon --exec \"npm test\"",
+    "build": "npm run build:clean && tsc",
+    "build:clean": "rimraf lib",
+    "test": "standard && ts-standard && npm run build && nyc mocha --exit --timeout 20000 tests/*test.js ",
```

Comment:
@jerome-benoit I guess we an just use ts-standard and remove the first standard command here ?

Reply:
Can we please do this in separate branches, otherwise it will be really chaotic here

Reply:
I tend to agree with @Shinigami92, ts-standard, even if based on standard seems to behave differently on javascript code. 
The goal of its introduction was to: 
+ keep a known working linter setup for all files except TS files;
+ add a similar linter setup to deal with TS files and standard misbehave with them.